Mathlib/RingTheory/PowerSeries/Inverse.lean

```diff
@@ -0,0 +1,239 @@
+/-
+Copyright (c) 2019 Johan Commelin. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Johan Commelin, Kenny Lau
+-/
+
+import Mathlib.RingTheory.PowerSeries.Basic
+import Mathlib.RingTheory.MvPowerSeries.Inverse
+
+#align_import ring_theory.power_series.basic from "leanprover-community/mathlib"@"2d5739b61641ee4e7e53eca5688a08f66f2e6a60"
+
+/-! # Formal power series - Inverses
+
+If the constant coefficient of a formal power series is invertible,
+then this formal power series is invertible.
+
+Formal power series over a local ring form a local ring.
+
```

Comment:
One could either add here the same implementation note that I suggested for MV ones concerning inverses, or point to that comment here.